src/xcode/ENA/ENA/Source/Developer Menu/DMDeveloperMenu.swift

```diff
@@ -15,10 +15,10 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#if !RELEASE
```

Comment:
Why not just `DEBUG`?

Reply:
We still have other configurations like `Community`